Makefile

```diff
@@ -60,7 +60,7 @@ mypy:
 
 .PHONY: test
 test:
-	pytest --cov=pydantic
+	pytest -p "no:hypothesispytest" --cov=pydantic
```

Comment:
IMO it would be better to add it in `setup.cfg` with `addopts = -p no:hypothesispytest`
It will allow people to keep using directly `pytest`